rust-runtime/smithy-eventstream/fuzz/fuzz_targets/roundtrip.rs

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use smithy_eventstream::frame::Message;
+
+fuzz_target!(|message: Message| {
+    let mut buffer = Vec::new();
+    if message.write_to(&mut buffer).is_ok() {
```

Comment:
this fails only when the message is too long, right? would be good to have a little bit of sanity checking that the error case only happens when the buffer is too long